.github/oxid-esales/defaults/defaults_light.yml

```diff
@@ -0,0 +1,388 @@
+# Purpose
+# =======
+# This file defines the default settings for the test workflow. It contains all
+# variables. The variables can then be overridden in more specific yaml files
+# to run different test scenarios.
+#
+# Variables used in multiple steps will be defined in the "workflow" and "global"
```

Comment:
nope, there are two sections, one is named "workflow" and the other is named "global". You won't find a line like
```
global sections:
```
below, because it contains a space which is a bad idea to have in variable names. But you'll find a line "workflow:" and a line "global:" which contain a lot of variables.